File: src/main/java/namingservice/replies/LocalizationReply.java
package namingservice.replies;

import requestreplyapi.entries.ExtendedEntry;

public class LocalizationReply extends Reply {
    private ExtendedEntry entry_data;

    public LocalizationReply(String reply_type, ExtendedEntry entry_data, boolean request_resolved) {
        super(reply_type, request_resolved);
        this.entry_data = entry_data;
    }

    public ExtendedEntry getEntry_data() {
        return entry_data;
    }

    public void setEntry_data(ExtendedEntry entry_data) {
        this.entry_data = entry_data;
    }

    @Override
    public String toString() {
        return "LocalizationReply{" +
                "entry_data=" + entry_data +
                '}';
    }
}

File: ObjectRequestBrokerCS/ORB/namingservice/replies/LocalizationReply.cs
using namingservice.replies;
using ORB.requestreplyapi.entries;

namespace ORB.namingservice.replies
{
	using ExtendedEntry = ExtendedEntry;

	public class LocalizationReply : Reply
	{
		private ExtendedEntry entry_data;

		public LocalizationReply(string reply_type, ExtendedEntry entry_data, bool request_resolved) : base(reply_type, request_resolved)
		{
			this.entry_data = entry_data;
		}

		public LocalizationReply()
		{
		}

		public virtual ExtendedEntry Entry_data
		{
			get
			{
				return entry_data;
			}
			set
			{
				this.entry_data = value;
			}
		}


		public override string ToString()
		{
			return "LocalizationReply{" +
					"entry_data=" + entry_data +
					'}';
		}
	}

}